src/components/VideoCard/VideoCard.component.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+import { useHistory } from 'react-router';
+
+import {
+  Card,
+  CardContainer,
+  CardImage,
+  CardContent,
+  Title,
+  Description,
+} from './VideoCard.styles';
+
+export default function VideoCard({ video: { id, description, image, title } }) {
+  const { theme } = useContext(ThemeContext);
+  const history = useHistory();
+
+  const openVideoPage = () => {
+    history.push(`/watch/${id}`);
+  };
+
+  return (
+    <Card role="listitem" theme={theme} onClick={openVideoPage}>
```

Comment:
For SEO purposes it's a good practice to use `<Link to={yourURL}></Link>` instead of `onClick={() => history.push(yourURL)}`. This is not critical, but if you want you can wrap your `Card` component with`<Link to={...}>`to achieve this.